drivers/hmis/app/graphql/schema.graphql

```diff
@@ -367,6 +367,11 @@ enum Component {
   """
   HORIZONTAL_GROUP
 
+  """
+  Render contents in an info box
+  """
+  INFO_GROUP
```

Comment:
This is sort of TBD, it's something Ash is toying with for displaying data quality inputs in visually distinct boxes. Once we decide if/how we're going to use it, maybe I can come up with a better name.